docs/change_controlled.rst

```diff
@@ -0,0 +1,82 @@
+This is the usual workflow for controlled documents generated from the SysML model where the content is developed in an external tool and only exported to latex periodically.
+
+Since neither ``draft`` nor ``master`` can be rewritten, it is important that if branches are created off of ``master`` and the content is still present in ``draft``, that ``master`` be merged back into the ``draft`` branch to ensure that ``draft`` is using a consistent version of the baseline.
+Given this requirement, it is preferred that ``draft`` be used in the usual case.
+Regardless of which base branch is used, ticket branches should follow standard :ref:`DM branching policies <git-branching>`.
+
+Once a document handle has been issued a repository can be created in the ``lsst`` GitHub organization using a repository name that matches the handle.
+If the document exists in a pre-existing repository, it should be renamed to the handle and, if necessary, moved to the correct organization.
+
+.. warning::
+  It may be necessary to adjust the history of a pre-existing repository if it has been developed using different policies, in particular it is important that ``master`` be adjusted such that it only contains merge commits from approved versions (and the initial placeholder commit).
+
+All documents use a Travis configuration file to build the document and deploy it to the corresponding https://ldm-nnn.lsst.io/ web site.
+Built PDFs of documents should not be committed to the repository.
+They are deployed automatically to the web during development and the final version will also be archived on DocuShare.
+
+During development:
+
+* Use the ``lsstdraft`` class option to make it clear that the document has not been approved.
+* Specify a date using :code:`\today` so that the built document continues to have an updated date for each change (we do not automatically insert a date based on the state of the git repository).
+* Update the change record but do not specify a version number.
+  The version number will be added during the release process.
+* New references should be added to a local bib file and that file should be added to the bibliography search path.
+* If the document is receiving minor edits consider using the :code:`\newtext` and :code:`\oldtext` macros to indicate text changes.
+  For larger changes the :command:`latexdiff-vc` command may be helpful when preparing for an RFC or change request submission.
+
+.. note::
+  Should draft mode automatically override the release date?
+
+When a document is ready to be reviewed by the CCB, it should be uploaded to DocuShare.
+The review process is more efficient if the changes to be reviewed are marked clearly on the PDF.
+If the CCB process triggers new modifications these should be made on a branch named ``tickets/RFC-nnn`` or ``tickets/LCR-nnn`` (relative to ``draft``) to make clear that the work was triggered by the review process.
```

Comment:
Why isn't this work ticketed in the normal way (ie, `DM-NNNN`)? It seems like something we should track through our regular process.